### ISSUE TITLE
Plain and Paragraph at end of document

### ISSUE THREAD

**Issue by kysko** [jgm/lunamark#76]
Not sure if this is intended or a bug.

If a multiblock document ends with a text line (only Inlines) not followed by a new line, that last line is interpreted as a Plain, not a Paragraph.

More generally, here are some results when applying `parse_blocks` to the following strings:

```
s1 = "A\n\nB"     -- A will be a Para, B will be a Plain
s2 = "A\n\nB\n"   -- A will be a Para, B will be a Plain
s3 = "A\n\nB\n\n" -- A will be a Para, B will be a Para
```

It seems to me that B should be a Para for s1 and s2, like in s3.

If however this is as intended, please disregard.

**Comment by Omikhleia** [comment]
Might also relate to #63 -- the workaround there too is to inject a few blank lines at the end of the input.

**Comment by kysko** [comment]
True, but for my needs I'm more interested in the general case with `parse_blocks`. I should have titled better, which I'll do right away.

For now, my workaround is:

- remove trailing blanklines;
- check if there's a blank line in the string
- If there is one, add two trailing blanks to force a Para;
- if there's none, do nothing to keep the Plain.

It's good enough for now.
